lib/bap_powerpc/bap_powerpc_cpu.mli

```diff
@@ -0,0 +1,40 @@
+open Bap.Std
+open Bap_powerpc_rtl
+open Bap_powerpc_dsl
+
+type cpu = {
```

Comment:
it's the third copy of the record definitions, please move it to a separate module, e.g., `bap_powerpc_types.ml` that has only type definitions and thus do not require an mli file.